.github/workflows/slovensko_digital_ci.yml

```
@@ -0,0 +1,21 @@
+name: Slovensko.Digital CI
+
+on: 
+  push:
+    branches:
+      - master
```

toto asi vieme skratit takto

```suggestion
    branches: 'master'
```